src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -120,9 +120,7 @@ <h5><span jhiTranslate="artemisApp.studentExams.participationStatus"></span></h5
                             @if (!studentExam.submitted) {
                                 <span jhiTranslate="entity.action.toggleToSubmitted"></span>
                             } @else {
-                                @if (studentExam.submitted) {
-                                    <span jhiTranslate="entity.action.toggleToUnsubmitted"></span>
-                                }
+                                <span jhiTranslate="entity.action.toggleToUnsubmitted"></span>
```

Comment:
Was this intentional?

Reply:
Yes, I do regard 
```
  @if (!studentExam.submitted) {
      <span jhiTranslate="entity.action.toggleToSubmitted"></span>
  } @else {
      <span jhiTranslate="entity.action.toggleToUnsubmitted"></span>
  }
```

as a cleaner solution than the current implementation
```
  @if (!studentExam.submitted) {
      <span jhiTranslate="entity.action.toggleToSubmitted"></span>
  } @else {
      @if (studentExam.submitted) {
          <span jhiTranslate="entity.action.toggleToUnsubmitted"></span>
      }
  }
```

This seems to be a leftover from the angular migration, and no ngElse was used in this place. Do I overlook something?

Reply:
I did not check the context of the code, just saw something that seem to not fit into the test case visibilty changes. This makes sense!